plugins/qlcplus_4.12.1/presets.js

```diff
@@ -0,0 +1,1002 @@
+  },
+  RampUpFastToSlow: {
+    isApplicable: cap => exportHelpers.isShutterEffect(cap, `RampUp`) && exportHelpers.isDecreasingSpeed(cap),
+    importCapability: () => importHelpers.getShutterStrobeCap(`RampUp`, `fast`, `slow`)
+  },
+  RampDownSlowToFast: {
+    isApplicable: cap => exportHelpers.isShutterEffect(cap, `RampDown`) && exportHelpers.isIncreasingSpeed(cap),
+    importCapability: () => importHelpers.getShutterStrobeCap(`RampDown`, `slow`, `fast`)
+  },
+  RampDownFastToSlow: {
+    isApplicable: cap => exportHelpers.isShutterEffect(cap, `RampDown`) && exportHelpers.isDecreasingSpeed(cap),
+    importCapability: () => importHelpers.getShutterStrobeCap(`RampDown`, `fast`, `slow`)
+  },
+
+
+  // color capabilities
+
+  ColorMacro: {
+    isApplicable: cap => (cap.type === `ColorPreset` || cap.isSlotType(`Color`)) && cap.colors !== null && cap.colors.allColors.length === 1,
+    exportRes1: cap => cap.colors.allColors[0],
+    importCapability: capData => capabilityPresets.ColorDoubleMacro.importCapability(capData)
+  },
+  ColorDoubleMacro: {
+    isApplicable: cap => (cap.type === `ColorPreset` || cap.isSlotType(`Color`)) && cap.colors !== null && cap.colors.allColors.length === 2,
+    exportRes1: cap => cap.colors.allColors[0],
+    exportRes2: cap => cap.colors.allColors[1],
+    importCapability: ({ channelName, res1, res2, index }) => {
+      if (channelName.match(/wheel\b/i)) {
+        return {
+          type: `WheelSlot`,
+          slotNumber: index + 1
+        };
+      }
+
+      const colors = [res1];
+      if (res2) {
+        colors.push(res2);
+      }
+
+      return {
+        type: `ColorPreset`,
+        comment: ``,
+        colors
+      };
+    }
+  },
+  ColorWheelIndex: {
+    isApplicable: cap => cap.type === `WheelRotation` && cap.wheels[0].type === `Color` && exportHelpers.isRotationAngle(cap),
+    importCapability: () => ({
+      type: `WheelRotation`,
+      angleStart: `0deg`,
+      angleEnd: `360deg`,
+      helpWanted: `Are these the correct angles?`
+    })
+  },
+
+
+  // gobo capabilities
+
+  // TODO: import/export a gobo image as res1
+  GoboShakeMacro: {
+    isApplicable: cap => cap.type === `WheelShake` && (cap.isSlotType(/Gobo|Iris|Frost/) || cap.isSlotType(`Open`)),
+    exportRes1: cap => (cap.isSlotType(`Open`) ? `Others/open.svg` : null),
+    importCapability: ({ capabilityName, index }) => {
+      const cap = {
+        type: `WheelShake`,
+        slotNumber: index + 1
+      };
+
+      const comment = importHelpers.getSpeedGuessedComment(capabilityName, cap);
+
+      if (`speedStart` in cap) {
+        cap.shakeSpeedStart = cap.speedStart;
+        cap.shakeSpeedEnd = cap.speedEnd;
+        delete cap.speedStart;
+        delete cap.speedEnd;
+      }
+
+      cap.comment = comment;
+
+      return cap;
+    }
+  },
+  GoboMacro: {
+    isApplicable: cap => cap.isSlotType(/Gobo|Iris|Frost/) || cap.isSlotType(`Open`),
```

Comment:
```suggestion
    isApplicable: cap => cap.type === `WheelSlot` cap.isSlotType(/Gobo|Iris|Frost/) || cap.isSlotType(`Open`),
```

To exclude `WheelSlotRotation`